client/app/(authenticated)/bceidbusiness/industry_user/dashboard/select-operator/(request-access)/user-operator/[id]/[formSection]/page.tsx

```diff
@@ -3,7 +3,7 @@ import SelectOperatorRequestAccessReceivedUserOperatorPage from "@/app/component
 export default async function Page({
   params,
 }: {
-  readonly params?: Readonly<{ id: number }>;
+  readonly params?: Readonly<{ id: number | string; formSection: number }>;
```

Comment:
Added type `string` for `id` so we can use `create` in the same route which simplifies a lot of code similar to the operations form.

Reply:
Isn't this how we do it in the `Operations` form? We reuse the same routes but instead of passing a `0` we pass `create` and then do a check before attempting to fetch the data if it isn't a valid id eg: `create`.

Reply:
Sorry but I made it `create` based in the operations page instead of 0 based on your feedback from a previous PR, which is also why I refactored this to be the same. Feel free to make a tech debt card.